### ISSUE TITLE
Pipeline cutting off paths from inputfile argument

### ISSUE THREAD

**Issue by Pallavi-Vankayala** [icbi-lab/quanTIseq#13]
Hello,
I am having issues with the quanTIseq pipeline with Singularity. I have included an input file with accurate file paths for my FASTQ files but I keep getting the following error: 

Starting quanTIseq pipeline with Singularity
 not found! Check input files!

I have tested the "test" data and command line that was included on https://icbi.i-med.ac.at/software/quantiseq/doc/ webpage, and that seemed to run smoothly but I am having difficulty with replicating the same with my data.

**Comment by chripla** [comment]
Hello, 

can you please provide us your inputfile as well as the file structure and the command you are executing? 

Thanks,
Christina

**Comment by Pallavi-Vankayala** [comment]
Thanks for your quick response!

In terms of the inputfile file structure, I have created a tab-delimited file **without** headers. I have created the inputfile within the same directory as the FASTQ files. 

The following is the command that I have run:
`bash quanTIseq_pipeline.sh --inputfile=inputfile.txt --outputdir=.`

[inputfile.txt](https://github.com/icbi-lab/quanTIseq/files/7951649/inputfile.txt)
